src/Sql/Sql/help/Set-AzSqlServerAuditing.md

```diff
@@ -15,66 +15,156 @@ Changes the auditing settings of an Azure SQL server.
 ```
 
-### Example 2: Disable the auditing policy of an Azure SQL server
+### Example 2: Disable the blob storage auditing policy of an Azure SQL server
 ```
 PS C:\>Set-AzSqlServerAuditing -State Disabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01"
 ```
 
-### Example 3: Enable the auditing policy of an Azure SQL server using a storage account from a different subscription
+### Example 3: Enable the blob storage auditing policy of an Azure SQL server using a storage account from a different subscription
 ```
 PS C:\>Set-AzSqlServerAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -StorageAccountName "Storage22" -StorageAccountSubscriptionId "7fe3301d-31d3-4668-af5e-211a890ba6e3"
 ```
 
-### Example 4: Enable the auditing policy of an Azure SQL server with advanced filtering using a T-SQL predicate.
+### Example 4: Enable the blob storage auditing policy of an Azure SQL server with advanced filtering using a T-SQL predicate
 ```
 PS C:\>Set-AzSqlDatabaseAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -StorageAccountName "Storage22" -DatabaseName "Database01" -PredicateExpression "statement <> 'select 1'"
 ```
 
-### Example 5: Remove the advanced filtering setting from the auditing policy of an Azure SQL server.
+### Example 5: Remove the advanced filtering setting from the blob auditing storage policy of an Azure SQL server
 ```
 PS C:\>Set-AzSqlDatabaseAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -StorageAccountName "Storage22" -DatabaseName "Database01" -PredicateExpression ""
 ```
 
+### Example 6: Enable the event hub auditing policy of an Azure SQL server
+```
+PS C:\>Set-AzSqlServerAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -EventHubName "EventHubName" -EventHubAuthorizationRuleResourceId "EventHubAuthorizationRuleResourceId"
+```
+
+### Example 7: Disable the event hub auditing policy of an Azure SQL server
+```
+PS C:\>Set-AzSqlServerAuditing -State Disabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -EventHubName
+```
+
+### Example 8: Enable the log analytics auditing policy of an Azure SQL server
+```
+PS C:\>Set-AzSqlServerAuditing -State Enabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -LogAnalytics -WorkspaceResourceId "/subscriptions/4b9e8510-67ab-4e9a-95a9-e2f1e570ea9c/resourceGroups/insights-integration/providers/Microsoft.OperationalInsights/workspaces/viruela2"
+```
+
+### Example 9: Disable the log analytics auditing policy of an Azure SQL server
+```
+PS C:\>Set-AzSqlServerAuditing -State Disabled -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -LogAnalytics
+```
+
+### Example 9: Disable, through pipeline, the log analytics auditing policy of an Azure SQL server
+```
+PS C:\>Get-AzSqlServer -ResourceGroupName "ResourceGroup01" -ServerName "Server01" | Set-AzSqlServerAuditing -LogAnalytics -State Disabled
+```
+
```

Comment:
Please add piping scenario examples

Reply:
Piping scenario examples exist already in Test-AuditingOnServer and Test-AuditingOnDatabase tests.

Reply:
They already exist.